### DISCUSSION TITLE
Unfortunately we stll confuse with global color settings at screen viewer

### DISCUSSION THREAD

**Discussion by joyshmitz** [riclolsen/json-scada#14] [Q&A] [answered]
Unfortunately we stll confuse with global color settings at screen viewer, I did uncomment for needed paths at C:\json-scada\src\htdocs\conf\config_viewers.js files, and change them accordingly to our needs, but all screen is still in steelblue color, i use correct color variables in my svg, only with uncommented strings in config file, there is no bpoblem. But configuration is ignored, I alsy try to change them at template config file, but still no effect... Where we are wrong? Maybe need to on something art php config file, or others?

**Comment by riclolsen** [comment, marked as answer]
The template file is just to recreate original config files, it does not affect anything.
After you change the config file be sure to reload files avoiding cache by using CTRL-SHIFT-R in chrome/chromium (or F12 and forced reload).
If the problem persists please send the SVG file for me to check.

**Reply by joyshmitz** [reply]
worked, thanks, we will remember about the cache
